DOCS/man/input.rst

```diff
@@ -2702,30 +2702,6 @@ Property list
     enabled, or after precise seeking). Files with imprecise timestamps (such
     as Matroska) might lead to unstable results.
 
-``window-scale`` (RW)
```

Comment:
is removed from docs intentionally?

Reply:
Yeah it's mostly redundant/outdated information. Like the other options, it should be in `DOCS/man/options.rst`